daemon/fuzz_test.go

```diff
@@ -0,0 +1,288 @@
+//go:build linux
+
+package daemon // import "github.com/docker/docker/daemon"
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/moby/sys/mount"
+	"github.com/sirupsen/logrus"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/backend"
+	imagetypes "github.com/docker/docker/api/types/image"
+	"github.com/docker/docker/container"
+	"github.com/docker/docker/daemon/config"
+	"github.com/docker/docker/daemon/images"
+	"github.com/docker/docker/image"
+	dockerreference "github.com/docker/docker/reference"
+	"github.com/opencontainers/go-digest"
+
+	fuzz "github.com/AdaLogics/go-fuzz-headers"
```

Comment:
Nit: can you group these imports in a single group? (we split our imports into 2 groups: `stdlib` and "other")